osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
@AndreasVIRES All objects have the same reference point. Independent of ego or not. What is causing confusion here is:
A position vector consists of two points:
start point: This is defined by the reference frame. (in OSI: sensor-frame or world frame)
end point: often referred to as reference point. It is **always** the middle of the bounding box. This is not depending on being an ego vehicle or not, and not depending on the coordinate system (reference frame).

Moreover a vector always is dependent of the direction in which it is regarded. This is also captured by the reference frame.

In other words: inconsistencies between ego and other vehicles do not exist.

I will capture this in a wiki article and refer to it here.